### DISCUSSION TITLE
Supported Processors

### DISCUSSION THREAD

**Discussion by LogicGirlUSA** [openvinotoolkit/openvino#25893] [General]
My computer has
AMD A6-3420M APU with Radeon(tm) HD Graphics 1.50 GHz
will this be added to the list of supported cpu's in the future?

**Comment by ilya-lavrenov** [comment]
We don't perform official validation on AMD processors, hence we don't add it to the list of supported HW.

But it should work as it's x86 architecture as Intel CPUs.

**Reply by LogicGirlUSA** [reply]
Thanks for your reply.
When I ran Audacity with openvino installed, I received an error; Divide By Zero Exception, so it crashed.
Any suggestions?
